hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMKeyRenameRequestWithFSO.java

```diff
@@ -119,14 +119,13 @@ public void testValidateAndUpdateCacheWithFromKeyInvalid() throws Exception {
 
   @Test
   public void testPreExecuteWithUnNormalizedPath() throws Exception {
+    addKeyToTable(fromKeyInfo);
     String toKeyName =
         "///root" + OzoneConsts.OZONE_URI_DELIMITER +
             OzoneConsts.OZONE_URI_DELIMITER +
             UUID.randomUUID();
     String fromKeyName =
-        "///root/sub-dir" + OzoneConsts.OZONE_URI_DELIMITER +
-            OzoneConsts.OZONE_URI_DELIMITER +
-            UUID.randomUUID();
+        "///" + fromKeyInfo.getKeyName();
```

Comment:
Just curious is there any specific reason behind this change?

Reply:
This test runs `OMKeyRenameRequestWithFSO.preExecute` with unnormalize paths.
With ACL checks moved to `preExecute`, the fromKeyName is now expected to be an existing one. Previously it was not a requirement and that's why fromKeyName could be generated from a `UUID.randomUUID()`.

Reply:
Got it. Thanks @duongkame .